addons/mrp/views/mrp_routing_views.xml

```diff
@@ -26,7 +26,7 @@
                                 <field name="name"/>
                                 <field name="workcenter_id" context="{'default_company_id': company_id}"/>
                                 <field name="sequence" groups="base.group_no_one"/>
-                                <field name="bom_id" readonly="1"/>
+                                <field name="bom_id" required="1" invisible="context.get('default_bom_id')"/>
```

Comment:
I have a little issue with the required="1", If you create a new BoM (don't save it yet), you cannot create operation on it (also the bom_id field will be visible because id is not set, it is for this reason that in master a use a custom context key `bom_id_invisible`).